app/scripts/lib/createDupeReqFilterMiddleware.ts

```diff
@@ -0,0 +1,86 @@
+import { JsonRpcMiddleware } from 'json-rpc-engine';
+import log from 'loglevel';
+
+const SECOND = 1000;
```

Comment:
[nit] this already exists [here](https://github.com/MetaMask/metamask-extension/blob/develop/shared/constants/time.ts)